### PR TITLE
feat(mux): Add Wezterm plugin for easier setup on Wezterm (experimental)

### LINKED ISSUE #143: Use wezterm.plugin.require() for easy install on wezterm
### Similar Issues

- [X] Before filing, I have searched for similar issues.

### Description

The installation process on wezterm could be improved by utilising `western.plugin.require()` to reduce clutter in a users config

### PR DESCRIPTION
**Pull request by mrjones2014**
Fixes #143 

Note: this functionality is still experimental.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -346,7 +346,30 @@ bind-key -T copy-mode-vi 'C-\' select-pane -l
...
+local smart_splits = wezterm.plugin.require('https://github.com/mrjones2014/smart-splits.nvim')
+local config = wezterm.config_builder()
+-- you can put the rest of your Wezterm config here
+smart_splits.apply_to_config(config, {
+  -- the default config is here, if you'd like to use the default keys,
+  -- you can omit this configuration table parameter and just use
+  -- smart_splits.apply_to_config(config)
+
+  -- directional keys to use in order of: left, down, up, right
+  direction_keys = { 'h', 'j', 'k', 'l' }
```
Comment (MLFlexer) [inline README.md:363]: You are missing a comma at the end of this line, otherwise it wont parse

**Comment by MLFlexer** [comment]
I have tested this in wezterm version: `wezterm 20230712-072601-f4abf8fd` on NixOS, by following the readme, and then checking out the branch manually, as it is not possible to checkout branches with `wezterm.plugin.require()`. However the functionality should be the same.

It is working as expected! :smile:

**Review by MLFlexer — CHANGES_REQUESTED** [review]
You should change the readme to parsable lua as per the comment, It looks good otherwise

File: README.md (hunk 1 above)
Comment (mrjones2014) [reply README.md:363]: Whoops, good catch, thanks

**Pull request merged**